account_invoice_report_hls/report/invoice_delivery.xml

```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="external_layout_invoice_hls">
+        <div
+            class="header"
+            t-att-style="report_header_style"
+        >
+            <!-- <div class="row">
+                <div class="col-3 mb4">
+                    <img
+                        t-if="company.logo"
+                        t-att-src="image_data_uri(company.logo)"
+                        style="max-height: 45px;"
+                        alt="Logo"
+                    />
+                </div>
+            </div> -->
+            <!-- <div
+                t-if="company.logo or company.report_header"
+                class="row zero_min_height"
+            >
+                <div class="col-12">
+                    <div style="border-bottom: 1px solid black;" />
+                </div>
+            </div> -->
+            <!-- <div class="row">
+                <div t-if="doc_type != 'invoice'" class="col-12 text-right">
+                    <strong t-field="o.invoice_id.date_invoice" t-options='{"widget": "date"}' />
+                </div>
+            </div> -->
+        </div>
```

Comment:
```suggestion
```
Any reason to still keep this part?

Reply:
I thought I might need the header later , but I don't need one.